ngrok.html

```diff
@@ -269,20 +268,20 @@ <h3>Properties</h3>
         </ul>
     </p>
     <p><code>Binding</code> - bind an HTTPS or HTTP endpoint or both. Expected values are <code>["http", "https"]</code>, <code>["http"]</code>, or <code>["https"]</code>. It is <b>STRONGLY</b> recommended to use HTTPS. Only valid when protocol is "http"</p>
-    <p><code>Domain</code> - allows you to specify a subdomain e.g. setting this to fred will use reserved tunnel name https://fred.ngrok.io, you can also enter a FQDN if that has been configured on ngrok NOTE: This option requires a paid subscription to ngrok. Only valid when protocol is "http"</p>
-    <p><code>Host header</code> - allows you to rewrite the HTTP Host header to this value. Only valid when protocol is "http"</p>
+    <p><code>Domain</code> - allows you to specify a subdomain (1 free) e.g. setting this to fred will use reserved tunnel name https://fred.ngrok.io, you can also enter a FQDN if that has been configured on ngrok NOTE: This option requires a paid subscription to ngrok. Only valid when protocol is "http"</p>
```

Comment:
PM from ngrok here. The domain field should take an FQDN, as subdomain has been deprecated. We have added a free domain to all ngrok accounts that you can claim when you sign up, which means this can be used with a free account as well.

Reply:
thanks fixed subdomain is used as domain, for backward compatibility